acceptance/internal/server.go

```diff
@@ -152,6 +152,18 @@ func AddHandlers(server *testserver.Server) {
 		return req.Workspace.JobsCreate(request)
 	})
 
+	server.Handle("POST", "/api/2.2/jobs/create", func(req testserver.Request) any {
+		var request jobs.CreateJob
+		if err := json.Unmarshal(req.Body, &request); err != nil {
+			return testserver.Response{
+				Body:       fmt.Sprintf("internal error: %s", err),
+				StatusCode: 500,
+			}
+		}
+
+		return req.Workspace.JobsCreate(request)
+	})
+
 	server.Handle("POST", "/api/2.0/pipelines", func(req testserver.Request) any {
```

Comment:
Also for the 2.1 version of the GET API (`/api/2.1/jobs/get`), can it be removed?